tests/shared/src/test/scala/scala/meta/tests/tokenizers/Scala3TokensSuite.scala

```diff
@@ -0,0 +1,23 @@
+package scala.meta.tests.tokenizers
+
+import munit.FunSuite
+import scala.meta.internal.tokenizers.ScalametaTokenizer
+import scala.meta.Dialect
+import scala.meta.inputs.Input.{File => MetaFile}
+import scala.meta.io.AbsolutePath
+import java.io.File
+import scala.meta.internal.parsers.ScalametaParser
+
+class Scala3TokensSuite extends FunSuite {
+
+  val testDir = "/home/kpbochenek/vl/github/official/dotty-example-project/src/main/scala"
```

Comment:
is this portable?

Reply:
:sweat_smile: was testing it for ` lampepfl/dotty-example-project` and forgot to remove it, thanks for spotting it :+1: